lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.html

```diff
@@ -5,8 +5,8 @@
                              [expanded]="canExpandProperties()"
                              [attr.data-automation-id]="'adf-metadata-group-properties'">
             <mat-expansion-panel-header>
-                <mat-panel-title role="heading">
```

Comment:
Why did you changed heading role to h1? Wouldn't be enough just to add aria-level in this case?

Reply:
Yes, you are right that this is also a solution for that. There is reason why I decided to use h tag instead and if you think I should to use aria-level then let me know. That reason is described in following pages: 
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-level
"Opt for using the <h1> thru <h6> elements instead."

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/heading_role
"This defines the text in the <div> to be the main heading of the page, indicated by being level 1 via the aria-level attribute. Opt for using the <h1> thru [<h6>] element instead."

And also interesting explaining here, please read first answer: 
https://webmasters.stackexchange.com/questions/137159/for-seo-is-role-heading-aria-level-1-the-same-as-h1

Please let me know what do you think :)

Reply:
Ok I reviewed the stuff you linked and I think this is fine :D Approved